src/com/tekion/validator/UserInputValidator.java

```diff
@@ -0,0 +1,12 @@
+package com.tekion.validator;
+
+public class UserInputValidator {
```

Comment:
this has no usages.
Also this is a utility class. pls make the constructor private